collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt

```diff
@@ -28,6 +28,8 @@ class FirstLaunchActivity : CollectAbstractActivity() {
 
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setTheme(R.style.Theme_Collect_Light)
```

Comment:
This initially fixed the issue, but I ended up also disabling the orientation lock as well. It feels like this activity should use one of our themes though to prevent any other issues so don't see a reason not to include this change.